bigbluebutton-html5/imports/ui/components/nav-bar/styles.scss

```diff
@@ -51,7 +51,7 @@
     bottom: 2px;
     right: 3px;
     background-color: $color-danger;
-    border: 2px solid;
+    border: 2px solid $color-gray-dark;
```

Comment:
could we change usage of pixel values to variables?

Reply:
I did not understand what you want say

Reply:
border: $some-variable solid $color-gray-dark;